backend/entities/AccessRequest.cs

```diff
@@ -44,14 +44,14 @@ public class AccessRequest : BaseEntity
         /// get - the list of agencies that the user is requesting to be added to.
         /// </summary>
         /// <returns></returns>
-        public ICollection<Agency> Agencies { get; set; } = new List<Agency>();
+        public ICollection<AccessRequestAgency> Agencies { get; set; } = new List<AccessRequestAgency>();
```

Comment:
Don't bother including `set;`.  It's not a desired way to interact with entity collections.